services/teams/components/team-selection.tsx

```diff
@@ -5,5 +5,11 @@ export async function TeamSelection() {
 	const allTeams = await fetchUserTeams();
 	const currentTeam = await fetchCurrentTeam();
 
-	return <TeamSelectionForm allTeams={allTeams} currentTeam={currentTeam} />;
+	return (
+		<TeamSelectionForm
+			allTeams={allTeams}
+			currentTeam={currentTeam}
+			key={currentTeam.dbId}
```

Comment:
:memo: In the following link, a key prop has been added so that the data is updated immediately with the redirect process. 

https://github.com/giselles-ai/giselle/blob/8e3ffa28c8d1cbd5dc92e643ca3570627bdc57b5/app/(main)/settings/team/actions.ts#L547